From b81618e8fed30c77ecbac5225790a50a7209aa89 Mon Sep 17 00:00:00 2001
From: Han Gao <rabenda.cn@gmail.com>
Date: Thu, 20 Mar 2025 00:07:39 +0800
Subject: [PATCH 427/552] config: net: disable realtek phy for eic7700x

disable realtek PHY, not working after turning it on

working:
[    3.367988] mdio_bus stmmac-0:00: attached PHY driver [unbound] (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
[    3.685222] mdio_bus stmmac-1:00: attached PHY driver [unbound] (mii_bus:phy_addr=stmmac-1:00, irq=POLL)
[    3.694720] mdio_bus stmmac-1:01: attached PHY driver [unbound] (mii_bus:phy_addr=stmmac-1:01, irq=POLL)
[   13.203352] win2030-eth-dwmac 50400000.ethernet end0: PHY [stmmac-0:00] driver [Generic PHY] (irq=POLL)
[   13.292339] win2030-eth-dwmac 50410000.ethernet end1: PHY [stmmac-1:00] driver [Generic PHY] (irq=POLL)

not work:
[    3.479834] RTL8211F Gigabit Ethernet stmmac-0:00: attached PHY driver (mii_bus:phy_addr=stmmac-0:00, irq=POLL)
[    3.803086] RTL8211F Gigabit Ethernet stmmac-1:00: attached PHY driver (mii_bus:phy_addr=stmmac-1:00, irq=POLL)
[    3.813200] RTL8211F Gigabit Ethernet stmmac-1:01: attached PHY driver (mii_bus:phy_addr=stmmac-1:01, irq=POLL)
[   13.469200] win2030-eth-dwmac 50400000.ethernet end0: PHY [stmmac-0:00] driver [RTL8211F Gigabit Ethernet] (irq=POLL)
[   13.681203] win2030-eth-dwmac 50410000.ethernet end1: PHY [stmmac-1:00] driver [RTL8211F Gigabit Ethernet] (irq=POLL)

Signed-off-by: Han Gao <rabenda.cn@gmail.com>
---
 arch/riscv/configs/win2030_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/riscv/configs/win2030_defconfig b/arch/riscv/configs/win2030_defconfig
index 33c92223468f..db1039b3d49a 100644
--- a/arch/riscv/configs/win2030_defconfig
+++ b/arch/riscv/configs/win2030_defconfig
@@ -515,7 +515,6 @@ CONFIG_NETXEN_NIC=m
 CONFIG_QED=m
 CONFIG_QEDE=m
 # CONFIG_NET_VENDOR_QUALCOMM is not set
-CONFIG_R8169=y
 # CONFIG_NET_VENDOR_RENESAS is not set
 # CONFIG_NET_VENDOR_ROCKER is not set
 # CONFIG_NET_VENDOR_SAMSUNG is not set
-- 
2.50.1

